SwrveSDK.podspec

```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |s|
   s.dependency 'SwrveSDKCommon', '4.5.2'
   s.dependency 'SwrveConversationSDK', '4.5.2'
 
-  s.frameworks = 'UIKit', 'QuartzCore', 'CFNetwork', 'StoreKit', 'Security', 'CoreTelephony', 'MessageUI', 'CoreLocation', 'AddressBook', 'AVFoundation', 'AssetsLibrary', 'Contacts', 'Photos'
+  s.frameworks = 'UIKit', 'QuartzCore', 'CFNetwork', 'StoreKit', 'Security', 'CoreTelephony', 'MessageUI', 'CoreLocation', 'AddressBook', 'AVFoundation', 'AssetsLibrary'
+  s.weak_frameworks = 'Contacts', 'Photos'
```

Comment:
it's described in our integration doc: https://docs.swrve.com/developer-documentation/integration/ios/